test/graaflib/algorithm/strongly_connected_components/kosaraju_test.cpp

```diff
@@ -0,0 +1,53 @@
+#include <graaflib/algorithm/strongly_connected_components/kosaraju.h>
+#include <gtest/gtest.h>
+
+namespace graaf::algorithm {
+
+TEST(KosarajuTest, EqualOutputSCCS) {
+  using sccs_t = std::vector<std::vector<vertex_id_t>>;
```

Comment:
It would be nice to have some more tests here, but I would be fine with doing this in a follow-up. Maybe you could reuse some of the predefined graph scenarios in `test/utils/scenarios/scenarios.h`.

Reply:
That makes sense - I think based on the code coverage I also could add a test on what happens if a blank graph is passed in. 

I will do up the tests for the ones in `scenarios.h`, are there any other ones that you possibly had in mind?

Reply:
Sounds good to me! I would also add the graph setup for which the algorithm currently fails. Any follow up scenarios we could also do in a follow-up!